jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-client/src/main/java/org/jbpm/console/ng/cm/client/util/CaseActionStatus.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.console.ng.cm.client.util;
+
+import java.util.Arrays;
+import java.util.List;
+
+public enum CaseActionStatus {
+
+    READY("Ready"),
+    INPROGRESS("Reserved", "InProgress"),
+    COMPLETED_SUCCESS("Completed"),
+    COMPLETED_FAIL("Suspended", "Failed", "Error", "Exited", "Obsolete"),
+    ADHOC_FRAGMENT("AdhocFragment");
+
+    private final List<String> statuses;
+
+    CaseActionStatus(final String... statuses) {
+        this.statuses = Arrays.asList(statuses);
+    }
+
+    public static CaseActionStatus fromStatus(final String status) {
+        for (CaseActionStatus status1 : CaseActionStatus.values()) {
+            if (status1.getStatus().contains(status)) {
+                return status1;
+            }
+        }
+        return null;
+    }
+
+    public List<String> getStatus() {
```

Comment:
I believe the purpose of this class is to "collapse" task statuses into simpler set of case statuses. If that is the case we should make it more clear when we're talking about Case status and when about task status. In particular I'd rename 1) statuses to taskStatuses and 2) getStatus to getTaskStatuses because it returns a list of values, not just 1 value.
Also it wouldn't hurt to make statuses immutable list `this.statuses = Collections.unmodifiableList(Arrays.asList(statuses))`